src/index.js

```diff
@@ -5,14 +5,17 @@ import App from './App';
 import store from './store'
 import { PersistGate } from 'redux-persist/integration/react';
 import { persistStore } from 'redux-persist';
+import { GoogleOAuthProvider } from '@react-oauth/google'
 
 let persistor = persistStore(store);
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
     <Provider store={store}>
         <PersistGate persistor={persistor}>
-            <App />
+            <GoogleOAuthProvider clientId='351717751939-dnrbo41t6vh74scajcho04sv9jmvb0vq.apps.googleusercontent.com'>
```

Comment:
You should set clientId in env

Reply:
Hi @phantuananh18 , `clientId` I can't set it from env . If you set clientId in env, it will fail in callback with GoogleOAuthProvider,.
if you have a better suggestion. please help me . Thank you.